js/controller.js

```diff
@@ -312,8 +312,21 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
       this.state.screenToShow = CONSTANTS.SCREEN.INITIAL_SCREEN;
     },
 
+    /**
+     * Set style "touch-action: none" only for video 360 on mobile devices
+     * see details: https://stackoverflow.com/questions/42206645/konvajs-unable-to-preventdefault-inside-passive-event-listener-due-to-target-be
+     */
+    setInlineStyles: function () {
+      if(this.videoVr && this.state.isMobile) {
+        this.state.mainVideoInnerWrapper.attr('style', 'touch-action: none');
+      } else {
+        this.state.mainVideoInnerWrapper.attr('style', '');
```

Comment:
can we do something like
`if (this.videoVr) {
     if (this.state.isMobile) {
        this.state.mainVideoInnerWrapper.attr('style', 'touch-action: none');
     } else {
         this.state.mainVideoInnerWrapper.attr('style', '');
     }
  }
`

Reply:
fix